core/src/main/kotlin/com/exactpro/th2/read/db/app/Requests.kt

```diff
@@ -29,7 +29,7 @@ class ExecuteQueryRequest(
 
 class PullTableRequest(
     val dataSourceId: DataSourceId,
-    val initQueryId: QueryId,
+    val initQueryId: QueryId?,
```

Comment:
Please, modify the gRPC implementation as well. It should correctly process the pulling task request without init query set